modules/comment/comment.service.js

```diff
@@ -21,7 +28,11 @@ class CommentsService {
   }
 
   async getAllCommentsByUser(userEmail) {
-    return Comment.find({ 'user.email': userEmail });
+    const comments = await Comment.find({ 'user.email': userEmail });
+    if (!comments.length) {
+      throw new Error(COMMENT_NOT_FOUND);
+    }
```

Comment:
I don't think that when the user has no comments yet we should display an error, a lot of users doesn't write comments at all, also this case is already handled by frontend

Reply:
fixed